config/default.json

```diff
@@ -2,7 +2,11 @@
   "bindAddress": "0.0.0.0",
   "mongo": {
     "url": "mongodb://localhost/openhim",
-    "atnaUrl": "mongodb://localhost/openhim"
+    "atnaUrl": "mongodb://localhost/openhim",
+    "connectionOptions": {},
+    "openHIMApiReadPreference": "primaryPreffered",
+    "openHIMApiWriteConcern": "1",
+    "openHIMApiReadConcern": "local"
```

Comment:
nvm i think it does, the docs points to certain options but this list thats present in mongodb should be allowed
http://mongoosejs.com/docs/connections.html#connection-string-options